src/views/CloneRepo.vue

```diff
@@ -1,20 +1,66 @@
 <template>
   <ViewWrapper title="Clone down a repository">
-    <div>
+    <label for="repo-url-input">
+      <strong>
+        Enter url:
+      </strong>
+    </label>
+    <div class="repo-url-select">
+      <input v-model="url" class="repo-url-input" placeholder="Enter url" />
+      <button @click="clone">Clone</button>
     </div>
+    <ReposFolderPicker label="Enter location:" />
```

Comment:
After playing with it locally, I think the default names that were already in the component are more clear as to what the folder this input field represents mean.

Which means we can remove the prop and revert a lot of the changes in that component.